core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -901,8 +901,9 @@ class PartKeyLuceneIndex(ref: DatasetRef,
   //scalastyle:on method.length
   def partIdsFromFilters(columnFilters: Seq[ColumnFilter],
                          startTime: Long,
-                         endTime: Long): debox.Buffer[Int] = {
-    val collector = new PartIdCollector() // passing zero for unlimited results
+                         endTime: Long,
```

Comment:
Can you remove the limit from this  and all the partId collectors ? This specific method is used in regular PromQL queries and it should not silently limit. The data could be aggregated

Reply:
Never mind - the default value for that param I see is Int.MaxValue, and PromQL API path is not passing in any value.